table_foosball/table_foosball_test.rb

```diff
@@ -9,50 +9,41 @@ def setup
     @kenichi = Player.new("Kenichi")
   end
 
+  def test_create_new_player
+    frederik = Player.new("Frederik")
+    assert_equal "Frederik", frederik.name
+    assert_empty frederik.games
+  end
+
```

Comment:
Your other project here is that Game state is leaked between tests. You'll want to clear ALL state between tests. The `teardown` helper can do that for you:

```ruby
def teardown
  Game.games = []
end
```